### ISSUE TITLE
Cannot find local zerotier-one server that doesn't listen on port 9993

### ISSUE THREAD

**Issue by bolu61** [zerotier/zeronsd#196]
Trying to start zeronsd on a host where zerotier-one server doesn't listen on port 9993 fails with the following message:
```
Aug 22 04:37:07.328  INFO zeronsd::init: Welcome to ZeroNS!
Error: Error: Communication Error error sending request for url (http://127.0.0.1:9993/network/<network-id>): error trying to connect: tcp connect error: Connection refused (os error 111). Are you joined to <network-id>?
```

**Comment by bolu61** [comment]
There is also no way to provide the actual port that zerotier-one listens on as a parameter. Or maybe it's just not shown in docs?

**Comment by laduke** [comment]
It looks like it's hard coded. I'm sure we could make it take a port or url as an argument. Just need to find time.

**Comment by erikh** [comment]
I'll try whacking this out tonight.

**Comment by erikh** [comment]
This is resolved as of #207. Please wait for that patch to be merged before trying zeronsd.

Hope this helps!

**Comment by erikh** [comment]
Also, your avatar has a most adorable kitten. :)
